src/components/Layout/Buttons/BaseButton.tsx

```diff
@@ -8,6 +8,7 @@ export interface BaseButton {
   borderColor?: ViewStyle["borderColor"];
   disabled?: boolean;
   fullWidth?: boolean;
+  iconOnly?: boolean;
```

Comment:
May not need this, unlikely we'll have icon only buttons since they aren't very recognisable/understandable